### ISSUE TITLE
Attention Mask not set

### ISSUE THREAD

**Issue by dlin95123** [codebasics/llm-fine-tuning-crash-course#1]
The cell with "from unsloth.chat_templates import get_chat_template" on its first line has the following warning/error message: The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.

I ran the notebook on my colab and received the above message. It would be great if you could fix it. Thanks.
